<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EqualizerMenu</class>
 <widget class="QDialog" name="EqualizerMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>472</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Equalizer Menu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QWidget" name="buttonRow" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>128</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QPushButton" name="resetGainsButton">
        <property name="text">
         <string>Reset Gains</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newPresetButton">
        <property name="text">
         <string>New Preset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="presetSelectLabel">
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Presets</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="presetSelect">
        <property name="maxVisibleItems">
         <number>10</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertPolicy::InsertBeforeCurrent</enum>
        </property>
        <item>
         <property name="text">
          <string>Select Preset</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hip-hop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>R&amp;B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Deep</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Speech</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vocal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Piano</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Jazz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rock</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Metal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Classic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alternative</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Techno</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Drum'n'Bass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Electronic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bass Reducer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bass Booster</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Full Treble</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Full Bass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Loudness</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Indie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reggae</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ska</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Full Bass &amp; Treble</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bandSelectLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Bands</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="bandSelect">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="firstSliderRow" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>196</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="firstSliderLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="secondSliderRow" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>196</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="secondSliderLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="enableButtonRow" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="enableButton">
        <property name="text">
         <string>Disabled</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
